### PR TITLE
Fixing tab tokens and updating variables.

### PR DESCRIPTION
**Pull request by gjones**
### Summary
No real visible changes here, but this PR fixes an issue where the border was being defined incorrectly.

### CODE REVIEW

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -30,21 +28,21 @@ const Trigger = styled(RadixTabs.Trigger)`
...
-    border-bottom: 2px solid ${props => props.theme.click.tabs.basic.stroke.active};
+    border-bottom: 2px solid ${props => props.theme.click.tabs.basic.color.stroke.active};
   }
 `;
 
 const Content = styled(RadixTabs.Content)``;
 
 const TriggersList = styled(RadixTabs.List)`
-  border-bottom: 1px solid ${props => props.theme.click.global.color.stroke.default};
+  border-bottom: 2px solid ${props => props.theme.click.global.color.stroke.default};
```
Comment (vineethasok) [inline src/components/Tabs/Tabs.tsx:45]: Why are we changing the border of the list of trigger container?

File: src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -30,21 +28,21 @@ const Trigger = styled(RadixTabs.Trigger)`
   }
 
   &:hover {
-    border-bottom: 2px solid ${props => props.theme.click.tabs.basic.stroke.hover};
-    background-color: ${props => props.theme.click.tabs.basic.color.background.hover};
```
Comment (vineethasok) [inline src/components/Tabs/Tabs.tsx:34]: background is removed

File: src/components/Tabs/Tabs.tsx (hunk 1 above)
Comment (vineethasok) [reply src/components/Tabs/Tabs.tsx:45]: Also should we add the display type as flex here

File: src/components/Tabs/Tabs.tsx (hunk 2 above)
Comment (gjones) [reply src/components/Tabs/Tabs.tsx:34]: What was the background colour got moved into the border-bottom attribute, so the background is still present.

Comment (gjones) [reply src/components/Tabs/Tabs.tsx:34]: Sorry, just re-read your comment, fixed.

**Review by vineethasok — APPROVED** [review]

**Pull request merged**